generators/server/templates/_travis.yml

```diff
@@ -46,6 +52,18 @@ script:
 <%_ } else if (buildTool === 'gradle') { _%>
   - ./gradlew bootRepackage -Pprod -x test
 <%_ } _%>
+  - docker-compose up -d
```

Comment:
This command won't work, because there is no `docker-compose.yml` file at the root folder.
It's because we migrate all files related to docker into src/main/docker/

Anyway, this line should be more complex:
- test type of database (mysql, postgresql, etc.)
- test if it's gateway or microservice, if so, need to start the registry too